page-supporters.php

```diff
@@ -33,33 +33,34 @@
                                      'id'=>'5','linkedin'=>'https://www.linkedin.com/company/dogpatch-labs',
                                      'twitter'=>'https://twitter.com/dogpatchlabs'],
 
-                             ['src'=>'../wp-content/themes/cd-theme/static/img/supporters/chq.png', 'name'=>'CHQ Building', 'jobDescription'=>'CHQ hosts & retains the CoderDojo team in Dogapatch Labs.',
+                             ['src'=>'../wp-content/themes/cd-theme/static/img/supporters/chq.png', 'name'=>'CHQ Building', 'jobDescription'=>'Bank of Ireland are growth partners across the island of Ireland, they are committed to supporting and starting Dojos around the country through their branch network',
```

Comment:
Were you meant to replace all of CHQ Building, or should BOI be a new entry? I'm guessing a new entry?